src/components/Button/styledComponents.tsx

```diff
@@ -0,0 +1,87 @@
+import styled, { css } from "styled-components";
+
+import {
+  activeButton,
+  hoverButton,
+  disabledButton,
+  hoverSecondaryButton,
+} from "../../assets/styles/colors";
+import { ReactComponent as Download } from "../../assets/img/icons/download.svg";
+import { minDevice } from "../../assets/styles/breakpoints";
+
+interface ButtonStyleProps {
```

Comment:
How about moving it to the interface file? Is this making any sense?

Reply:
Yes, good point :) I've moved it there.